airbyte-integrations/connectors/destination-mssql-v2/src/main/kotlin/io/airbyte/integrations/destination/mssql/v2/MSSQLChecker.kt

```diff
@@ -0,0 +1,14 @@
+package io.airbyte.integrations.destination.mssql.v2
+
+import io.airbyte.cdk.load.check.DestinationChecker
+import io.airbyte.integrations.destination.mssql.v2.config.DataSourceFactory
+import io.airbyte.integrations.destination.mssql.v2.config.MSSQLConfiguration
+import jakarta.inject.Singleton
+
+@Singleton
+class MSSQLChecker(private val dataSourceFactory: DataSourceFactory) : DestinationChecker<MSSQLConfiguration> {
+    override fun check(config: MSSQLConfiguration) {
```

Comment:
nonblocking, at some point we should add something to the CDK for this. Destination `check` should be running a sync to a `airybte_internal.test_stream_<random>` stream + dropping the resulting table, unless we have a good reason to do something different/additional.

Reply:
Updated